src/components/Dialog/Dialog.component.tsx

```diff
@@ -72,13 +72,15 @@ export const Dialog = (props: DialogProps) => {
 			className={`mm-dialog ${className}`}
 			{...restProps}
 		>
-			<DialogHeader>
-				<IconButton onClick={onCloseHandler}>
+			<DialogHeader style={{
+				...(!title && {justifyContent: 'flex-end'})
```

Comment:
Use props or utility class.
In cases where we use `styled` and `utility` classes together, all the defined utility classes should use `!important`;

Reply:
Created a prop